db/migrate/20180111192319_create_games.rb

```diff
@@ -0,0 +1,11 @@
+class CreateGames < ActiveRecord::Migration[5.0]
+  def change
+    create_table :games do |t|
+    	t.integer :user_id
+    	t.integer :opponent_id
+    	t.integer :current_turn_player_id
+    	t.integer :winning_player_id
```

Comment:
I think there should also be a "name" field for the game model.

Reply:
I'm not opposed to that, but what's the purpose behind it?

Reply:
I think we'll need the "name" field for when I add in the new game form. My trello card mentioned being able to add a new game with the single field "name.". @ilyakrasnov  Can you confirm if there should be a name field in the Game database?

Reply:
@eyoshimoriyama thats correct.. but this can be done via a migration when you work on that feature specifically..

Reply:
Ok will do!